starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -3038,7 +3064,7 @@ sub proseco_args {
             0 + $targ_cmd->{q3},
             0 + $targ_cmd->{q4}
         ],
-        man_angle => 0 + $targ_cmd->{angle},
+        man_angle => 0 + $man_angle,
```

Comment:
What a silly language with the `0 + something`.

Reply:
Some of this could probably go away -- I played some games here at one point to get floats over to Python but we could probably just be explicit on the Python side especially now that Inline::Python is gone.